ddl/db_test.go

```diff
@@ -5615,3 +5616,109 @@ func (s *testSerialDBSuite) TestColumnTypeChangeGenUniqueChangingName(c *C) {
 
 	tk.MustExec("drop table if exists t")
 }
+
+func (s *testSerialDBSuite) TestModifyColumnTypeWithWarnings(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+	tk.MustExec("use test")
+	// Enable column change variable.
+	tk.Se.GetSessionVars().EnableChangeColumnType = true
+	defer func() {
+		tk.Se.GetSessionVars().EnableChangeColumnType = false
+	}()
```

Comment:
We can remove those code? `tk.Se` will be useless when finishing this function?

Reply:
good